roles/rsyslog/tasks/inputs/ovirt/main.yml

```diff
@@ -1,18 +1,41 @@
 ---
 # Deploy configuration files
-- name: Install/Update oVirt logging packages and generate configuration files in /etc/rsyslog.d
+- name: Install/Update oVirt input packages and generate configuration files in /etc/rsyslog.d
   vars:
     __rsyslog_packages: "{{ __rsyslog_ovirt_prereq_packages + __rsyslog_ovirt_packages }}"
     __rsyslog_rules: "{{ __rsyslog_ovirt_rules }}"
-  include_role:
-    name: "{{ role_path | dirname | dirname | dirname }}"
-    tasks_from: deploy.yml
+  include_tasks:
+    file: deploy.yml
+
+- name: Create oVirt input configuration files in /etc/rsyslog.d
+  vars:
+    __rsyslog_input_name: "{{ ovirt_item.name }}"
+    __rsyslog_input_type: "{{ ovirt_item.type }}"
+    __rsyslog_packages: []
+    __rsyslog_rules:
+      - name: "input-ovirt-{{ ovirt_item.name }}"
+        type: input
+        nocomment: true
+        weight: "21"
+        sections:
+          - options: "{{ lookup('template', 'input_template.j2') }}"
+  include_tasks:
+    file: deploy.yml
+  loop: '{{ logging_inputs }}'
+  loop_control:
+    loop_var: ovirt_item
+  when: ovirt_item.type | d() == 'ovirt'
 
 - name: Allow rsyslog to listen on collectd port
   seport:
-    ports: "{{ rsyslog_read_collectd_port | d(44514) }}"
+    ports: "{{ ovirt_item.rsyslog_read_collectd_port | d(44514) }}"
     proto: tcp
     setype: syslogd_port_t
     state: present
-  when: 'ansible_selinux.mode in ["enforcing", "permissive"]'
+  loop: '{{ logging_inputs }}'
+  loop_control:
+    loop_var: ovirt_item
+  when:
+    - ovirt_item.type | d() == 'ovirt'
+    - 'ansible_selinux.mode in ["enforcing", "permissive"]'
```

Comment:
Why do you need the outer quotes (')?

Reply:
Probably, copy&paste error... :p Removing them.